### ISSUE TITLE
Add a doc for setting up AWS SSO

### ISSUE THREAD

**Issue by praneetloke** [pulumi/docs#1168]
...it would also be nice to have a doc that explains the various attributes that users are asked to configure.

**Comment by lukehoban** [comment]
@praneetloke Do you mean using AWS SSO with Pulumi, like as a fourth entry in https://www.pulumi.com/docs/guides/saml/?

**Comment by praneetloke** [comment]
> Do you mean using AWS SSO with Pulumi, like as a fourth entry in https://www.pulumi.com/docs/guides/saml/?

@lukehoban yep that's right. However, I do think the [SAML SSO overview doc](https://www.pulumi.com/docs/guides/saml/sso/) should help users figure out the values in the absence of an AWS specific configuration doc.
